### PR TITLE
Make WPS UI more reactive.

### PR DESCRIPTION
**Pull request by SPalominos**
Increase the speed of the Columns and Values listing in the WPS UI (issue #1228)

### CODE REVIEW

File: bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java [hunk 1]
```diff
@@ -117,7 +117,7 @@ void openProcess(URI processIdentifier,
      *
      * @return The list of the field name matching the accepted and excluded types.
```
Comment (ebocher) [inline bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java]: field name -> column name

File: bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java [hunk 2]
```diff
@@ -131,14 +131,13 @@ void openProcess(URI processIdentifier,
     List<String> getTableList(List<DataType> dataTypes, List<DataType> excludedTypes);
```
Comment (ebocher) [inline bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java]: columns

File: bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java [hunk 3]
```diff
@@ -131,14 +131,13 @@ void openProcess(URI processIdentifier,
...
 
     /**
-     * Returns a map containing field information like table type, SRID...
+     * Returns a list of maps containing field information like table type, SRID...
      *
      * @param tableName Name of the table.
-     * @param fieldName Name of the field.
      *
-     * @return Map containing the field information.
+     * @return List of map containing the field information.
```
Comment (ebocher) [inline bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java]: column

File: bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/impl/utils/ToolBoxIcon.java [hunk 4]
```diff
@@ -57,7 +57,7 @@
     public static final String GEO_FILE = "geofile";
     public static final String BROWSE = "browse";
     public static final String OPTIONS = "options";
-    public static final String JDBC_TABLE_FIELD = "jdbc_table_field";
+    public static final String JDBC_TABLE_COLUMN = "jdbc_table_field";
```
Comment (ebocher) [inline bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/impl/utils/ToolBoxIcon.java]: JDBC_TABLE_COLUMN = "jdbc_table_field" to JDBC_COLUMN = "jdbc_column";

File: bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/impl/utils/ToolBoxIcon.java [hunk 5]
```diff
@@ -57,7 +57,7 @@
...
     public static final String ENUMERATION = "enumeration";
     public static final String JDBC_VALUE = "jdbc_table_field_value";
```
Comment (ebocher) [inline bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/impl/utils/ToolBoxIcon.java]: "jdbc_table_field_value"; to "jdbc_value";

File: bundles/wps/wps-service-orbisgis/src/main/java/org/orbisgis/wpsserviceorbisgis/OrbisGISWpsServer.java [hunk 6]
```diff
@@ -49,10 +49,10 @@
  */
 public interface OrbisGISWpsServer extends WpsServer {
 
-    String TABLE_SRID = "TABLE_SRID";
-    String TABLE_IS_SPATIAL = "TABLE_IS_SPATIAL";
-    String TABLE_DIMENSION = "TABLE_DIMENSION";
-    String GEOMETRY_TYPE = "GEOMETRY_TYPE";
+    String COLUMN_NAME = "COLUMN_NAME";
```
Comment (ebocher) [inline bundles/wps/wps-service-orbisgis/src/main/java/org/orbisgis/wpsserviceorbisgis/OrbisGISWpsServer.java]: Why not an enum ?

File: bundles/wps/wps-service-orbisgis/src/main/java/org/orbisgis/wpsserviceorbisgis/OrbisGISWpsServer.java [hunk 7]
```diff
@@ -89,7 +88,7 @@
      * @param excludedTypes
      * @return The list of the field name.
```
Comment (ebocher) [inline bundles/wps/wps-service-orbisgis/src/main/java/org/orbisgis/wpsserviceorbisgis/OrbisGISWpsServer.java]: field to column

File: bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java (hunk 2 above)
Comment (SPalominos) [inline bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java]: The error was in the doc, it has been fixed.

File: bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java (hunk 1 above)
Comment (SPalominos) [inline bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/api/OrbisGISWpsClient.java]: Fixed.

**Comment by SPalominos** [comment]
Changes done.

**Pull request merged**